### ISSUE TITLE
Please keep podspec in sync with tags

### ISSUE THREAD

**Issue by ReDetection** [stefanomondino/SMReactiveRestKit#3]
Hi there. 
It's been a while since #1 :) 
You have 0.4 in the cocoapods trunk and 0.5 in the repo. Well, it's easy to fix on my side but please keep it in sync

**Comment by stefanomondino** [comment]
You're right, but this time the problem seems to be RestKit, I'm not able to lint this library with cocoapods anymore, therefore i cannot synch with their trunk. I'm not using RestKit in my projects anymore, it was a really cool project at the beginning but I cannot afford to keep AFNetworking to version 1.3.3 (it's a strict dependency), iOS has evolved since then and they (RestKit team) seem not having noticed that. That's why I didn't update this library anymore, there are ALWAYS problems with RestKit, I've got enough (and also found something better for my taste -> RAC + Mantle + AFN2).

Can you help me fixing this lint problem? are you able to lint locally?

**Comment by ReDetection** [comment]
Well, I never tried actually. And I recently I used RestKit for some projects and now I'm getting strange crashes from their code very often, and their love to the main thread makes me sad a bit too.
So no, I'd rather keep using `:git` parameter in the podfile for now. Probably to reflect this in readme is a good idea. 
It's a bit offtopic but maybe you got any other library I should give a look?

**Comment by stefanomondino** [comment]
Actually, I should probably suggest to drop the entire library and move forward to something more...recent.

Personally, for objective-C projects, i'm using Reactive-Cocoa + AFNetworking 2.0+ Mantle. There's a SMReactiveRestKit equivalent for AFNetworking for Reactive functionalities. Mantle is made by the same great guy who partially wrote Reactive Cocoa and Carthage. I personally love the separation between network layer and modeling layer, I believe RK is wrong to mix them up in a single monolithic framework.
